jobs/jjb-deploy.yaml

```diff
@@ -20,6 +20,14 @@
           artifact-days-to-keep: 7
     dsl: |
       node {
+        stage('build-images') {
+          parallel jjb: {
+            sh "oc start-build --follow jjb"
```

Comment:
Use ' or " ?

Reply:
`'` would not expand variables. There are none, so it does not matter much here.